packages/apps/tools/src/scripts/graphCron/index.ts

```diff
@@ -67,32 +120,47 @@ export const runJobPerEnvironment = async (props: IEnvProps) => {
     }
 
     if (
-      Math.abs(totalHeightOnChainWeb - totalHeightOnGraph) >
+      Math.abs(totalHeightOnChainWeb - totalCutHeightGraph) >
       MAXBLOCKHEIGHT_DIFFERENCE
     ) {
       await sendErrorMessage({
-        title: `${props.env} graph issue`,
-        msg: `There is a large difference in the last blockheight between ${props.env} chainweb and graph. \n difference: ${Math.abs(totalHeightOnChainWeb - totalHeightOnGraph)}`,
+        title: `${props.env} chainweb and graph blockheight difference`,
+        msg: `There is a large difference in the totalCutHeight between ${props.env} chainweb and graph.
+
+Total cut height difference: ${Math.abs(totalHeightOnChainWeb - totalCutHeightGraph)}
+
+[Chainweb lastBlockHeight](${props.chainweb}): ${lastBlockHeightChainweb}
+[GraphiQL lastBlockHeight](${props.graphqlRef}): ${lastBlockHeightGraph}
+Total height difference: ${lastBlockHeightChainweb - lastBlockHeightGraph}
+`,
       });
     }
   } catch (e) {
     await sendErrorMessage({
       title: `There was a general issue with the ${props.env} graph cron job`,
-      msg: ``,
+      msg: e,
```

Comment:
are we sure this is a string? or an object? if object this will fail. maybe stringify to be sure?